build.zig

```diff
@@ -95,28 +97,32 @@ fn packagesCrossPlatform(b: *std.Build, options: Options) void {
     const target = options.target;
     const optimize = options.optimize;
 
-    zopengl_pkg = zopengl.package(b, target, optimize, .{});
     zmath_pkg = zmath.package(b, target, optimize, .{});
     zpool_pkg = zpool.package(b, target, optimize, .{});
     zglfw_pkg = zglfw.package(b, target, optimize, .{});
-    zsdl_pkg = zsdl.package(b, target, optimize, .{});
-    zmesh_pkg = zmesh.package(b, target, optimize, .{});
-    znoise_pkg = znoise.package(b, target, optimize, .{});
-    zstbi_pkg = zstbi.package(b, target, optimize, .{});
-    zbullet_pkg = zbullet.package(b, target, optimize, .{});
     zgui_pkg = zgui.package(b, target, optimize, .{
         .options = .{ .backend = .glfw_wgpu },
     });
     zgpu_pkg = zgpu.package(b, target, optimize, .{
         .options = .{ .uniforms_buffer_size = 4 * 1024 * 1024 },
         .deps = .{ .zpool = zpool_pkg.zpool, .zglfw = zglfw_pkg.zglfw },
     });
+    zems_pkg = zems.package(b, target, optimize, .{});
     ztracy_pkg = ztracy.package(b, target, optimize, .{
         .options = .{
-            .enable_ztracy = !target.isDarwin(), // TODO: ztracy fails to compile on macOS.
-            .enable_fibers = !target.isDarwin(),
+            .enable_ztracy = !target.isDarwin() and options.target.getOsTag() != .emscripten, // TODO: ztracy fails to compile on macOS.
```

Comment:
@michal-z is this comment still valid? I think we fixed it here: https://github.com/michal-z/zig-gamedev/pull/262?